### PR TITLE
Add bitwarden import tool

### PR DESCRIPTION
**Pull request by logic**
This is an initial pass at an importer for a bitwarden vault export; a few shortcomings were discovered with the CSV export format:

* Field types (text/hidden/boolean) are not exported (bitwarden/web#154)
* Field export is ambiguous due to hand-rolled serialization (bitwarden/web#154)
* Cards and Identities are not exported (bitwarden/web#148)

Still, this can be useful as a recovery-of-last-resort, and is useful for moving between instances where you don't necessarily have access to the backend database (for example, moving from vault.bitwarden.com to a self-hosted instance).

### CODE REVIEW

**Comment by Pschittt** [comment]
Hi,

Shouldn’t be more « useful » to implement the  « post "/ciphers/import" » function on the api, and let the web vault do the « transformation », if there is a self hosted web vault.

The web vault support already many formats.

**Comment by logic** [comment]
I agree that implementing the import API would be a useful addition; in this case, though, I was simply following along with the project's existing pattern of standalone tools for one-off importing (none of the existing tools, including the converter I wrote, do any sort of duplication checking, validation or data mangling beyond what's strictly necessary for the import, etc).

At a minimum, something like that would depend on getting a version of your PR (jcs/bitwarden-ruby#14) merged, which gets you a minimally-functional web UI. But not everyone is going to want to use the web vault, but will still want to be able to do a one-time conversion from whatever system they were using previously. So I think there will still be a need for CLI tooling here as well.

tl;dr: I think what you're asking for is probably best tracked as a separate issue that refactors common functionality in the conversion tooling into lib, adds error and duplication checks, and breaks out interfaces as REST APIs and CLI tooling. This is just a PR to make the basic conversion tool I already wrote available to others. 😃

**Comment by Pschittt** [comment]
You’re right. It makes totally sense.
